debian/ceph-base.install

```diff
@@ -3,6 +3,9 @@ lib/systemd/system/ceph-crash.service
 usr/bin/ceph-crash
 usr/bin/ceph-debugpack
 usr/bin/ceph-run
+usr/bin/cephfs-data-scan
```

Comment:
this is the `#9` case in https://wiki.debian.org/PackageTransition. shall we update the relation between ceph-base and ceph-common accordingly?